src/tables.rs

```diff
@@ -655,41 +651,17 @@ pub static dav1d_filter_2d: [[u8; RAV1D_N_FILTERS]; RAV1D_N_FILTERS] = [
     [0, 0, 0, FILTER_2D_BILINEAR as u8],
 ];
 
-pub static dav1d_filter_dir: [[u8; 2]; N_2D_FILTERS] = [
-    [
-        RAV1D_FILTER_8TAP_REGULAR as u8,
-        RAV1D_FILTER_8TAP_REGULAR as u8,
-    ],
-    [
-        RAV1D_FILTER_8TAP_SMOOTH as u8,
-        RAV1D_FILTER_8TAP_REGULAR as u8,
-    ],
-    [
-        RAV1D_FILTER_8TAP_SHARP as u8,
-        RAV1D_FILTER_8TAP_REGULAR as u8,
-    ],
-    [
-        RAV1D_FILTER_8TAP_REGULAR as u8,
-        RAV1D_FILTER_8TAP_SHARP as u8,
-    ],
-    [
-        RAV1D_FILTER_8TAP_SMOOTH as u8,
-        RAV1D_FILTER_8TAP_SHARP as u8,
-    ],
-    [RAV1D_FILTER_8TAP_SHARP as u8, RAV1D_FILTER_8TAP_SHARP as u8],
-    [
-        RAV1D_FILTER_8TAP_REGULAR as u8,
-        RAV1D_FILTER_8TAP_SMOOTH as u8,
-    ],
-    [
-        RAV1D_FILTER_8TAP_SMOOTH as u8,
-        RAV1D_FILTER_8TAP_SMOOTH as u8,
-    ],
-    [
-        RAV1D_FILTER_8TAP_SHARP as u8,
-        RAV1D_FILTER_8TAP_SMOOTH as u8,
-    ],
-    [RAV1D_FILTER_BILINEAR as u8, RAV1D_FILTER_BILINEAR as u8],
+pub static dav1d_filter_dir: [[Rav1dFilterMode; 2]; N_2D_FILTERS] = [
```

Comment:
The `#[no_mangle]` `static`s are accessed by assembly, so they can't change.  Ones like this that aren't can probably be changed to `const`s.  They were just translated as `static`s by `c2rust transpile` since they were `static`s in C.

Reply:
I made it a `const` now